beanie/odm/utils/init.py

```diff
@@ -21,6 +21,29 @@
 from beanie.odm.utils.relations import detect_link
 from beanie.odm.views import View
 
+ALL_TYPES_EXCEPT_NULL = [
```

Comment:
This needs an explanation in the inline comment with a reference to the doc to make it supportable. Or, if there is such a thing but with 0 in the pymongo package, it is better to construct this one based on it.

Reply:
This is a list of standard BSON type IDs: https://www.mongodb.com/docs/manual/reference/bson-types/ I'll add a link there.